.ci/azure-pipelines.yml

```
@@ -1,5 +1,8 @@
 variables:
-  - group: 'jellyfin'
+  group: 'jellyfin'
+  ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
+    - name: JELLYFIN_VERSION
+      value: replace(variables['Build.SourceBranch'], 'refs/tags/', '')
```

If I read the docks mentioned by @EraYaN right then you need to use this syntax here:
```suggestion
   JELLYFIN_VERSION: replace(variables['Build.SourceBranch'], 'refs/tags/', '')
```

---

I _think_ what Azure does is remove the if statement and place the yaml in the "body" where the if-statement was. I probably forgot to add back the "- " before the group definition. I'll try that first and otherwise test this change.

---

I might have fixed it now!

---

It doesn't work because I think Azure is setting the value to empty now..

---

u might need the expression syntax on the assignment sight tbh.
e.g. `JELLYFIN_VERSION: $[replace(variables['Build.SourceBranch'], 'refs/tags/', '')]`

---

Even if that is an issue, it should've build this PR as it is not building a tag right now, but I'll try

---

there is also [this](https://stackoverflow.com/a/57532526) xD

---

I mean it seems to work right now? no publish and a build with the 0.0.0-dev version as expected.